tox.ini

```diff
@@ -1,18 +1,20 @@
 [tox]
 
 envlist =
-    flake8, twistedchecker-diff
+    flake8
 
     # Twisted 15.5 is the first version to support Python 3 (3.3+)
-    trial-py{27,py,34,35,36}-tw{155,166,current,trunk}
+    coverage-py{27,py,34,35,36}-tw{155,166,current,trunk}
+
+    coverage_combine
 
     docs, docs-linkcheck
 
 skip_missing_interpreters = True
 
 
 ##
-# Default testenv
+# Build (default environment)
```

Comment:
I don't understand this comment change.  Arguably the comment should just be deleted since it doesn't say much, but this is not a "build" now, is it?  it's still a test env.  Builds are done with `setup.py wheel`.